code/__HELPERS/experimental.dm

```diff
@@ -19,7 +19,7 @@
 var/global/list/masterPool = new
 
 // Read-only or compile-time vars and special exceptions.
-var/list/exclude = list("inhand_states", "loc", "locs", "parent_type", "vars", "verbs", "type", "x", "y", "z","group")
+var/list/exclude = list("inhand_states", "loc", "locs", "parent_type", "vars", "verbs", "type", "x", "y", "z","group", "animate_movement")
```

Comment:
Also for those who aren't aware, the reason animate_movement has been added to this list is it has been resetting to 0 for some pooled objects like rods, metal, other things. So initial for that var is fucked or something, and anything that actually modifies animate movement wouldn't need resetting anyway.